README.md

```diff
@@ -39,7 +39,6 @@ prefer private disclosure, please email to all of the maintainers:
 
 - fpaoline@redhat.com
 - rbryant@redhat.com
-- rodrigoca@microsoft.com
```

Comment:
I think it's safe to remove @johananl from here too

Reply:
Good point, fixed! :)